src/dmd/dsymbol.d

```diff
@@ -1382,15 +1382,13 @@ extern (C++) class ScopeDsymbol : Dsymbol
     DsymbolTable symtab;        // members[] sorted into table
     uint endlinnum;             // the linnumber of the statement after the scope (0 if unknown)
 
-private:
     /// symbols whose members have been imported, i.e. imported modules and template mixins
     Dsymbols* importedScopes;
     Visibility.Kind* visibilities; // array of Visibility.Kind, one for each import
```

Comment:
Should probably change this to a DDoc comment.

Reply:
I'm going to create constant accessors to prevent modifying it by hand. I'm going to add documentation there.

Reply:
Well, a constant accessor is not possible, since they can be modified. I did the accessor part and added documentation. Can you re-review?

Reply:
> constant accessor is not possible, since they can be modified

I don't follow. Modified by whom? User of libdmd, or DMD itself?

Reply:
> I don't follow. Modified by whom? User of libdmd, or DMD itself?

They are pointers that are modifiable, so I need to explicitly cast it. The user can explicitly mutate it or DMD can do it by adding imports when running semantics, so casting it can't be trusted.